common/src/main/java/com/wynntils/services/map/MapService.java

```diff
@@ -50,6 +53,18 @@ public List<MapTexture> getMapsForBoundingCircle(BoundingCircle circle) {
                 .toList();
     }
 
+    public boolean isPlayerInMappedArea() {
+        MinimapOverlay minimapOverlay = Managers.Feature.getFeatureInstance(MinimapFeature.class).minimapOverlay;
+        float width = minimapOverlay.getWidth();
+        float height = minimapOverlay.getHeight();
+        float scale = minimapOverlay.scale.get();
+
+        BoundingCircle textureBoundingCircle = BoundingCircle.enclosingCircle(BoundingBox.centered(
+                (float) McUtils.player().getX(), (float) McUtils.player().getZ(), width * scale, height * scale));
+
+        return !getMapsForBoundingCircle(textureBoundingCircle).isEmpty();
+    }
```

Comment:
My last issue with this PR is this code. I don't like that we are referencing the overlay instance here, the whole method has a bit of "doesn't really fit the description" vibe. What if we simply check if the player x/y is not inside a mapped area? Or what if we just specify a constant box (eg. 100 in X-Y directions) and check that?

Reply:
Perhaps could just set the width, height and scale to the default values of the minimap overlay

Reply:
You know what? Just make it arguments of the function, and provide a default value..